18F/identity-idp
# Update Propshaft to 0.9.x

## 🛠 Summary of changes

Updates the [Propshaft](https://github.com/rails/propshaft) gem to the latest version.

See release notes: https://github.com/rails/propshaft/releases

Specifically, this is aimed at addressing two things:

- A cache invalidation issue we're noticing with desynchronized font URL references, hoping to have been fixed by https://github.com/rails/propshaft/pull/188 (v0.9.0)
- Remove the asset fingerprinting shortening monkey-patch added in #9272, as this appears to be the default as of https://github.com/rails/propshaft/pull/173 (v0.9.0)

## 📜 Testing Plan

1. Run `rails assets:precompile`
2. Run `rails s`
3. Go to http://localhost:3000
4. Observe no visual defects
5. Open developer tools Elements window
6. Observe assets (e.g. `<script>` and `<link>` tags) have digested URLs with a hash of 8 characters (e.g. 
 `/packs/validated_field_component-5f68c2d7.digested.js`)
